frontend/dist/index.mak

```diff
@@ -77,33 +77,33 @@
   <body>
     <div id='content'></div>
 
-    <script type='text/javascript' src='/app/tracim_frontend_lib.style.js'></script>
+    <script type='text/javascript' src='/app/tracim_frontend_lib.style.js?v=${build_version}'></script>
 
-    <script type='text/javascript' src='/assets/tracim/appInterface.js'></script>
-    <script type='text/javascript' src='/assets/tracim/tinymceInit.js'></script>
+    <script type='text/javascript' src='/assets/tracim/appInterface.js?v=${build_version}'></script>
+    <script type='text/javascript' src='/assets/tracim/tinymceInit.js?v=${build_version}'></script>
 
     % for custom_toolbox_file in custom_toolbox_files:
-    <script type='text/javascript' src='/custom_toolbox-assets/${custom_toolbox_file.name}'></script>
+    <script type='text/javascript' src='/custom_toolbox-assets/${custom_toolbox_file.name}?v=${build_version}'></script>
     % endfor
 
-    <script type='text/javascript' src='/assets/tracim.vendors~app.js'></script>
-    <script type='text/javascript' src='/assets/tracim.app.js'></script>
+    <script type='text/javascript' src='/assets/tracim.vendors~app.js?v=${build_version}'></script>
+    <script type='text/javascript' src='/assets/tracim.app.js?v=${build_version}'></script>
 
-    <script type='text/javascript' src='/app/workspace.app.js'></script>
+    <script type='text/javascript' src='/app/workspace.app.js?v=${build_version}'></script>
 
     % for app in applications:
-    <script type='text/javascript' src='/app/${app.minislug}.app.js'></script>
+    <script type='text/javascript' src='/app/${app.minislug}.app.js?v=${build_version}'></script>
     %endfor
-    <script type='text/javascript' src='/app/share_folder.app.js'></script>
-    <script type='text/javascript' src='/app/admin_workspace_user.app.js'></script>
-    <script type='text/javascript' src='/app/workspace_advanced.app.js'></script>
-    <script type='text/javascript' src='/app/gallery.app.js'></script>
+    <script type='text/javascript' src='/app/share_folder.app.js?v=${build_version}'></script>
+    <script type='text/javascript' src='/app/admin_workspace_user.app.js?v=${build_version}'></script>
+    <script type='text/javascript' src='/app/workspace_advanced.app.js?v=${build_version}'></script>
+    <script type='text/javascript' src='/app/gallery.app.js?v=${build_version}'></script>
 
-    <script type='text/javascript' src='/assets/bootstrap/jquery-3.2.1.js'></script>
-    <script type='text/javascript' src='/assets/bootstrap/popper-1.12.3.js'></script>
-    <script type='text/javascript' src='/assets/bootstrap/bootstrap-4.0.0-beta.2.js'></script>
+    <script type='text/javascript' src='/assets/bootstrap/jquery-3.2.1.js?v=${build_version}'></script>
+    <script type='text/javascript' src='/assets/bootstrap/popper-1.12.3.js?v=${build_version}'></script>
+    <script type='text/javascript' src='/assets/bootstrap/bootstrap-4.0.0-beta.2.js?v=${build_version}'></script>
 
```

Comment:
@Skylsmoi Does it be required on these javascript files (they have version in name)

Reply:
you're right, it isn't
If we change the library version, it'll be reflected in the filename

Reply:
fixed.